ci-operator/step-registry/hypershift/aws/destroy-nested-management-cluster/hypershift-aws-destroy-nested-management-cluster-chain.yaml

```diff
@@ -52,7 +53,8 @@ chain:
       --name="${CLUSTER_NAME}" \
       --namespace="${HYPERSHIFT_NAMESPACE}" \
       --infra-id=${CLUSTER_NAME} \
-      --base-domain=${DOMAIN}
+      --base-domain=${DOMAIN} \
+      --cluster-grace-period 40m
```

Comment:
based on [settings in hosted-cluster wf](https://github.com/openshift/release/blob/c0a7ee63273d5f4525534732d3831f736cb51294/ci-operator/step-registry/hypershift/hostedcluster/destroy/hostedcluster/hypershift-hostedcluster-destroy-hostedcluster-commands.sh#L41)